lms/djangoapps/discussion/tests/test_views.py

```diff
@@ -356,11 +356,11 @@ def setUp(self):
         # course is outside the context manager that is verifying the number of queries,
         # and with split mongo, that method ends up querying disabled_xblocks (which is then
         # cached and hence not queried as part of call_single_thread).
-        (ModuleStoreEnum.Type.mongo, 1, 6, 4, 15, 3),
-        (ModuleStoreEnum.Type.mongo, 50, 6, 4, 15, 3),
+        (ModuleStoreEnum.Type.mongo, 1, 5, 3, 13, 1),
```

Comment:
I believe it is because I consolidated the logic into a shared function, so it is no longer making duplicate queries. I also moved some of the queries into one side of the conditional rather than being executed for both. As you said, I felt the changes here were okay because the numbers were getting smaller, and the tests are all passing.